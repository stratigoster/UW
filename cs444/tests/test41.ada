PACKAGE foo IS
  procedure plus (a,b:string := "hi") return void;
--                                    ^^^^^^^^^^^^
-- procedures do not have return types
END foo;
