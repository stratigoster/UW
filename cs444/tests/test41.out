***ERROR***:2:39:	Expected ";" but found "return"
